README.md

```diff
@@ -0,0 +1,270 @@
+<a name="readme-top"></a>
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
```

Comment:
- [ ] Awesome adding a README file, but please kindly customize the sections; most importantly the **REQUIRED SECTIONS**. The Readme file is often the first file that the users read. It is a text file that contains information for the user about the software, project, or code, or it might contain instructions, help, or details about the patches or updates. 👍